src/main/webapp/scripts/captions.js

```diff
@@ -0,0 +1,20 @@
+
+function getCaptions() {
+  videoId = document.getElementById("video-url").value.split('v=')[1];
+  fetch(`http://video.google.com/timedtext?lang=en&v=${videoId}`)
+    .then(response => response.text())
+    .then((responseText) => {
+      const parser = new DOMParser();
```

Comment:
nit: maybe add a comment here to explain that this doesn't work with auto generated captions

Reply:
I'll be adding a block later on to check whether the user input URL is a valid URL/video has captions so it'll be obvious in the code and to the user later

Reply:
👌